### ISSUE TITLE
write @train_parameters

### ISSUE THREAD

**Issue by ZacCranko** [anj1/NeuralNets.jl#15]
to take each setting in a vector of train parameters and output a `Dict` for use in `TrainReport`, possibly with another `Dict` to convert variable names to user friendly names. e.g. `learning_rate` -> `"Learning rate"`.
